code/code/misc/ai_commands.cc

```diff
@@ -1006,9 +1006,6 @@ int TMonster::aiBow (TBeing *doer, TBeing *, aiTarg cond)
   if (!isHumanoid())
     return FALSE;
 
-  if (doer->getSex() == SEX_FEMALE)
-    doWhisper(format("%s Pssst, polite women curtsey...") % fname(doer->name));
-
```

Comment:
I am genuinely pretty offended by it, and have been ever since I first ran into it. You requested changes, but the request seems to actually be "don't merge", so I'm not sure what I'm supposed to do.

Reply:
Well, I'm not offended by it, I find it archaic and funny.  Though if you REALLY REALLY want to see it gone, my objection is not particularly strong.

Let's make a compromise, write a non-sexist funny mob response to replace this. It'll make me stop complaining you're ripping features out of the game.

Reply:
As the thing itself is inherently sexist, I don't believe there's any response I could substitute it with. I'll sleep on it though.

Reply:
Nah, you could add any cool response/action anywhere else. The more visible to players the better.

Reply:
well I can definitely think on that! I know I mentioned hairballs, but I'll figure out something that's actually a response